.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "MF-LBM-extFiles"]
 	path = MF-LBM-extFiles
-	url = https://github.com/lanl/MF-LBM-extFiles.git
```

Comment:
shall we move back to the absolute path? Does relative path work when someone forks the repo?

Reply:
Relative paths are actually better, as it will use the same protocol everywhere, i.e. everything goes over ssh or https.